### ISSUE TITLE
Awesome

### ISSUE THREAD

**Issue by Christophe-Gauge** [akkana/scripts#12]
Not an issue, I just wanted to thank you for the awesome code, and also wanted to let you know that I ported your code as an interactive iPython notebook that users can now run online:

https://github.com/Christophe-Gauge/NEOWISE/blob/master/comet.ipynb

**Comment by akkana** [comment]
Neat! That's the best argument for notebooks I've seen yet. I was going to write a followup blog entry today, so I'll add a link. And note that I'm still tweaking python.py to add features when I think of them: a couple of days ago I added an altitude table and twilight times, so you can see what times the comet is actually high enough to see with the sky dark enough.

**Comment by Christophe-Gauge** [comment]
Thank you for your reply. I changed the structure to refer to your Python script instead of duplicating the code. The only change I had to make is to point the cache folder to the local directory as ftp is not allowed.

**Comment by wrljet** [comment]
Yes, thanks for sharing.  This Pi/PuTTY/vi thing has bugged me for years.
